crates/exex/exex/src/dyexex.rs

```diff
@@ -0,0 +1,32 @@
+//! Dynamically loading type-safe abstractions
+
+/// Dynamically loaded ExEx entrypoint, that accepts the [`ExExContext`](`reth_exex::ExExContext`)
+/// and returns a Future that will be polled by the [`ExExManager`](`reth_exex::ExExManager`).
+///
+/// ## Example usage:
+/// ```rust
+/// use std::future::Future;
+/// use reth_exex::{define_exex, ExExContext};
+/// use reth_node_api::FullNodeComponents;
+///
+/// // Create a function to produce ExEx logic
+/// async fn exex<Node: FullNodeComponents>(_ctx: ExExContext<Node>) -> eyre::Result<()> {
+///     Ok(())
+/// }
+///
+/// // Use the macro to generate the entrypoint function
+/// define_exex!(exex,<Node>);
```

Comment:
do we need to pass the generic argument explicitly here? Can't we just hardcode it to `Node`, since it's not accessible to the user anyway?

Reply:
let me see

Reply:
we can do that, yep